# ML-02 — Research Question and Provisional Lane

This skeleton is yours to fill. Work the sections **in order** — each one has a one-line hint. Simple words, honest numbers.

> Working with an AI assistant? Tell it to read `skills/README.md` first and load the one skill this assignment names on its card.

## 1. My lane (or freestyle) and why

*Name your lane — or say 'freestyle' and describe your own question. One short paragraph: why this one?*

### My Lane

I selected **Refresh / Content Opportunity Scoring**.

I chose this lane because the starter pipeline already demonstrates a complete workflow for identifying content pages that deserve review. It uses observable search and engagement signals instead of product decisions and produces a ranked refresh queue with transparent reason codes. Since I successfully executed the starter pipeline, this lane provides a strong starting point for my capstone project.

## 2. The question: decision, action, cost of a wrong call

*What decision does your work improve? Who acts on it? What does a wrong recommendation cost?*

### Research Question

**Question**

Which content pages should be reviewed first for refresh based on observable search and engagement signals?

**Unit of Analysis**

One content page.

**Decision**

Identify which pages should be reviewed first.

**Action**

- Refresh
- Expand
- Monitor
- Review CTR

**Cost of a Wrong Recommendation**

A wrong recommendation may waste editor time, prioritize the wrong pages, or overlook pages that actually need attention.

**Why Data / ML Can Help**

Observable search and engagement signals can be combined to prioritize pages more effectively than simple manual rules.

## 3. Quick look at the data (2-3 real numbers)

*Load the starter CSV below and show 2-3 real numbers that make your lane look worth the next 7 weeks.*

In [6]:
import pandas as pd

df = pd.read_csv("../../data/raw/content_refresh_anonymized.csv")

print("Rows:", len(df))
print("Columns:", len(df.columns))
print("Average CTR:", round(df["ctr"].mean(), 4))
print("Average Impressions (90d):", round(df["impressions_90d"].mean(), 2))
print("Average Trend (%):", round(df["trend_pct"].mean(), 2))

Rows: 30000
Columns: 44
Average CTR: 0.5107
Average Impressions (90d): 5200.37
Average Trend (%): -4.79


### Quick Look at the Data

The starter dataset contains **30,000** content pages with **44** observable features.

A quick exploration shows:
- Average CTR: **0.5107**
- Average 90-day impressions: **5,200.37**
- Average trend percentage: **-4.79%**

These statistics suggest that the dataset contains meaningful search and engagement signals that can support a content refresh prioritization workflow.

## 4. Careful words: what I can and can't claim

*Write what your work will be able to say (observed, directional, decision-support) — and what it never will (causal proof, 'predicting Google').*

### Careful Words

This project provides decision-support recommendations based on observable search and engagement signals.

The model does not prove that refreshing a page will improve traffic or rankings. Other factors, such as seasonality, search intent changes, algorithm updates, or content consolidation, may also affect page performance.

The recommendations should be reviewed by humans before taking action.

## Self-check

Before you submit, confirm each line honestly:

- [ ] Every section above is filled — markdown thinking AND the code that backs it
- [ ] The notebook runs top to bottom with no errors (Runtime → Run all)
- [ ] No client names, URLs, or private queries anywhere
- [ ] My claims use careful words: observed, measured, directional, decision-support
- [ ] Committed to my repo under `work/notebooks/` — then submit your repo URL on the card. Done.

### Self Check

- [x] I selected one lane.
- [x] I defined a research question.
- [x] I identified the decision and action.
- [x] I included real numbers from the starter dataset.
- [x] I explained what I can and cannot claim.